src/components/StackedBarChart/drawD3Chart.tsx

```diff
@@ -125,26 +129,7 @@ export function drawD3Chart(
   // style y and x axis
   const yAxis = axisLeft(yScale).ticks(yTicks);
 
-  const xAxis = axisBottom<Date>(xScale)
-    .ticks(6)
-    .tickFormat((tickDate: Date): string => {
-      // get sysdate for x Axis
-      const today = new Date();
-
-      const dateIsToday =
-        tickDate.getFullYear() === today.getFullYear() &&
-        tickDate.getMonth() === today.getMonth() &&
-        tickDate.getDate() === today.getDate();
-
-      if (dateIsToday) {
-        return 'Heute';
-      }
-      // let formattedTime = formatter(d);
-      let daysBack = today.getTime() - tickDate.getTime();
-      daysBack = daysBack / (1000 * 3600 * 24);
-      daysBack = Math.round(daysBack);
-      return daysBack === 1 ? 'Gestern' : `Vor ${daysBack} Tagen`;
-    });
+  const xAxis = axisBottom<Date>(xScale).ticks(5).tickSizeOuter(0);
```

Comment:
Here the "Heute" that @vogelino mentioned goes missing. I'm also wondering if we should use the default tick format in general. The Leipzig team did a `26.04.` format which I think is good because it's short and precise

Reply:
Also, it would make the translation of the labels unecessary.